File: P12_u3.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.p12_u3;
import java.util.Scanner;
/**
 *
 * @author richy
 */
public class P12_u3 {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int c, total, num;
        System.out.println("Escribe el total de los numeros a procesar: ");
        total= sc.nextInt();
        for(c=1; c<=total; c++){
            System.out.println("Escribe los numeros: ");
            num= sc.nextInt();
            if (num%2 == 0)
                System.out.println("Par");
            else 
                System.out.println("Impar");
        } 
            
    }
}

File: p12_u3.cs
using System;

namespace p12_u3
{
    internal class Program
    {
        static void Main(string[] args)
        {
            int c, total, num;
            Console.WriteLine("Escribe el total de los numeros a procesar: ");
            total = Convert.ToInt32(Console.ReadLine());
            for (c = 1; c <= total; c++)
            {
                Console.WriteLine("Escribe los numeros: ");
                num = Convert.ToInt32(Console.ReadLine());
                if (num % 2 == 0)
                    Console.WriteLine("Par");
                else
                    Console.WriteLine("Impar");
            }
        }
    }
}
